streaming/python/tests/test_word_count.py

```diff
@@ -45,14 +46,23 @@ def sink_func(x):
                 (old_value[0], old_value[1] + new_value[1])) \
         .sink(sink_func)
     ctx.submit("word_count")
-    import time
-    time.sleep(3)
+
+    def check_succeed():
+        if os.path.exists(sink_file):
+            import time
+            time.sleep(3)  # Wait all data be written
```

Comment:
We don't need this anymore.

Reply:
We write file using append mode, so we need to wait a little for all data being written

Reply:
You basically only need to provide a function which simply returns true or false and let `wait_for_condition` handle the sleep and retry.

Reply:
fixed